samples/excel/89-preview-apis/style.yaml

```diff
@@ -0,0 +1,241 @@
+
+                // Apply new style.
+                worksheet.getRange("A1:E1").style = ("Diagonal Orientation Style");
+                worksheet.getRange("A1:E1").format.verticalAlignment = "Justify";
+
+                let range = worksheet.getRange("A1:E1");
+                context.load(range);
+                context.load(range.format);
+
+                await context.sync();
+            });
+        }
+
+        async function applyBuiltInStyle() {
+            await Excel.run(async (context) => {
+                let worksheet = context.workbook.worksheets.getItem("Sample");
+                context.load(worksheet);
+
+                // Apply built-in style. 
+                // Styles are in the Home tab ribbon.
+                worksheet.getRange("A1:E1").style = Excel.BuiltInStyle.neutral;
+                worksheet.getRange("A1:E1").format.horizontalAlignment = "Right";
+
+                let range = worksheet.getRange("A1:E1");
+                context.load(range);
+                context.load(range.format);
+
+                await context.sync();
+            });
+        }
+
+        async function getStyleFontProperties() {
+            await Excel.run(async (context) => {
+                let style = context.workbook.styles.getItem("Normal");
+                context.load(style);
+                style.font.load("bold, color, italic, name, size");
+                style.fill.load("color");
+
+                await context.sync();
+
+                console.log("Bold: " + style.font.bold);
+                console.log("Font color: " + style.font.color);
+                console.log("Italic: " + style.font.italic);
+                console.log("Name: " + style.font.name);
+                console.log("Size: " + style.font.size);
+                console.log("Fill color: " + style.fill.color);
+            });
+        }
+
+        async function getStyleAlignmentProperties() {
+            await Excel.run(async (context) => {
+                let style = context.workbook.styles.getItem("Diagonal Orientation Style");            
+                context.load(style);
+
+                await context.sync();
+
+                console.log("Orientation: " + style.orientation);
+                console.log("Horizontal alignment: " + style.horizontalAlignment);
+                console.log("Add indent: " + style.addIndent);
+                console.log("Reading order: " + style.readingOrder);
+                console.log("Wrap text: " + style.wrapText);
+                console.log("Include protection: " + style.includeProtection);
+                console.log("Shrink to fit: " + style.shrinkToFit);
+                console.log("Style locked: " + style.locked);
+            });
+        }
+
+        async function deleteNewStyle() {
+            await Excel.run(async (context) => {
+                let style = context.workbook.styles.getItem("Diagonal Orientation Style");
+                context.load(style);
```

Comment:
You don't need this load; because deleting an object doesn't require reading it back to the task pane.